docs/conversion_examples_gallery/recording/intan.rst

```diff
@@ -1,13 +1,19 @@
-Intan data conversion
+Intan Data Conversion
 ---------------------
 
+This guide covers the conversion of Intan data, including both amplifier data (primary neural recordings) and analog data (auxiliary inputs, ADC inputs, DC amplifiers) from RHD2000 and RHS2000 systems.
+
 Install NeuroConv with the additional dependencies necessary for reading Intan data.
 
 .. code-block:: bash
 
     pip install "neuroconv[intan]"
 
-Convert Intan data to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.intan.intandatainterface.IntanRecordingInterface`.
+Intan Amplifier Data Conversion
```

Comment:
This is my preference, so ignore if you think the original is the better fit.
```suggestion
Intan Recording Data Conversion
```

Reply:
The reason I choose this is that Recording is an spikeinterface object but all the data (both neural and non-neural) is loaded as a recording object so I wanted to disambiguate it. Within Intan parlance, the data that of this stream is called amplifier data and that's why I have chosen this term. What do you think?

Reply:
Ah thanks for pointing this out, in this case I'd go with your suggestion and drop mine.